prow/prowjobs/kubeflow/gcp-blueprints/kubeflow-gcp-blueprints.yaml

```diff
@@ -14,6 +14,8 @@ postsubmits:
   kubeflow/gcp-blueprints:
   - name: kubeflow-gcp-blueprints-postsubmit
     cluster: build-kubeflow
+    branches:
+    - ^master|.+-branch$
```

Comment:
nit: maybe this can be a bit more strict on what release branch looks like? 
If I wasn't looking at the PR title at the same time I would wonder what `-branch` means. So probably worth a comment at least. WDYT?

Reply:
Make sense, I have added a comment to explain v1.1-branch format in gcp-blueprints repo.